packages/@expo/cli/src/start/server/metro/withMetroMultiPlatform.ts

```diff
@@ -350,6 +413,42 @@ export function withExtendedResolver(
       };
     },
 
+    // Custom externals support
+    (context: ResolutionContext, moduleName: string, platform: string | null) => {
+      // We don't support this in the resolver at the moment.
+      if (moduleName.endsWith('/package.json')) {
+        return null;
+      }
```

Comment:
Do we need to support this eventually?

Reply:
As in, it's not really a TODO item

Reply:
We may need to support it in the future.